### PR TITLE
Reverted most of the changes made

### LINKED ISSUE #154: Multiple case numbers show up when the same case enters the court
## Task Description

- Multiple case numbers that are the same will raise errors when adding cases to the database due to its unique primary key constraint.
- We need to use a different key that encompasses instances of a court hearing rather than separate by case.

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
We changed our minds about how to approach duplicate case_no_ids -> we're no longer using this as a primary key.
Therefore, we no longer need to check whether a case number already exists in the database.

Only thing changed is the date regex.

Closes #154

### CODE REVIEW

**Comment by luciegamby-sigmalabs** [comment]
Closes #153

File: pipeline/extract.py [hunk 1]
```diff
@@ -36,16 +36,6 @@ def get_stored_titles(conn) -> list:
     return [row["title"] for row in result]
 
 
-def get_stored_case_number_ids(conn) -> list:
```
Comment (ErvinRex) [inline pipeline/extract.py:39]: Well done for removing unnecessary functions from the script

File: pipeline/extract.py [hunk 2]
```diff
@@ -150,7 +140,7 @@ def download_pdfs(court_case: dict) -> None:
         f.write(response.content)
 
 
-def parse_pdf(court_case: dict, case_nos: list):
+def parse_pdf(court_case: dict):
```
Comment (ErvinRex) [inline pipeline/extract.py:143]: Good catch so that errors don't arise for not passing enough arguments

**Review by ErvinRex — APPROVED** [review]
Glad we found a solution that would prevent us from dropping court cases, meaning more cases can be stored in the database.

**Pull request merged**